test/e2e/exec_test.go

```diff
@@ -0,0 +1,65 @@
+package integration
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Podman rmi", func() {
```

Comment:
Ditto this, this is the exec test but it looks like all rmi again?  Do I need more tea?

Reply:
fixed